### ISSUE TITLE
想请教一下在训练enlcn模型时，psnr值为nan

### ISSUE THREAD

**Issue by Zc229** [Zj-BinXia/ENLCA#15]
Preparing loss function:
1.000 * L1
[Epoch 1]       Learning rate: 1.00e-4

Evaluation:
0it [00:01, ?it/s]
[Set14 x4]      PSNR: nan (Best: nan @epoch 1)
Forward: 2.35s

**Comment by jiaweimmiao** [comment]
@Zc229 我也遇到了相同的问题，请问您的问题已经解决了吗？可以分享一下解决方案么？
